test/odbc/psqlodbc/psqlodbc_tests_common.cpp

```diff
@@ -0,0 +1,613 @@
+#include "psqlodbc_tests_common.h"
+
+// helper function to initialize insert string (1, "", "", ""), etc.
+string InitializeInsertString(const vector<string>& insertedValues, bool isNumericInsert, int pkStartingValue) {
+
+  string insertString{};
+  string comma{};
+  if (isNumericInsert) {
+    for (int i = 0; i < insertedValues.size(); i++) {
+      insertString += comma + "(" + std::to_string(pkStartingValue) + "," + insertedValues[i] + ")";
+      comma = ",";
+      pkStartingValue = pkStartingValue + 1;
+    }
+    return insertString;
+  }
+
+  for (int i = 0; i < insertedValues.size(); i++) {
+    if (insertedValues[i] == "NULL") {
+      insertString += comma + "(" + std::to_string(pkStartingValue) + "," + insertedValues[i] + ")";
+      comma = ",";
+    }
+    else {
+      insertString += comma + "(" + std::to_string(pkStartingValue) + ",\'" + insertedValues[i] + "\')";
+      comma = ",";
+    }
+    pkStartingValue = pkStartingValue + 1;
+  }
+  return insertString;
+}
+
+string createTableConstraint(const string &constraintType, const vector<string> &constrainCols) {
+  
+  string tableConstraints{constraintType + "("};
+  string comma{};
+  for (int i = 0; i < constrainCols.size(); i++) {
+    tableConstraints += comma + constrainCols[i];
+    comma = ",";
+  }
+  tableConstraints += ")";
+  
+  return tableConstraints;
+}
+
+void createTable(ServerType serverType, const string &tableName, const vector<pair <string, string>> &tableColumns, const string &constraints) {
+  
+  OdbcHandler odbcHandler(Drivers::GetDriver(serverType));
```

Comment:
This is more of a general comment. I guess, no need to act on it at the moment.
On one hand, this is very nice and self-contained.

The only issue could be a large number of extra database connect and disconnect, which are resource intensive.
The solution would be to pass the same odbcHandler reference to the functions.
But again, the current solution is nicer.
Only if the database connecting/disconnecting adds a long time to the test process we may think about it.